.claude/commands/tests/test_orchestrate.py

```
@@ -130,6 +130,103 @@ def test_redirect_functionality(self):
+            '--bead', 'bead-123',
+            '--validate', 'make test',
+            '--no-new-pr',
+            '--no-new-branch',
+            'My task description'
+        ]
+
+        with patch('sys.argv', test_args):
+            with patch('subprocess.run') as mock_run:
+                with patch('os.path.exists', return_value=True):
+                    mock_run.return_value.returncode = 0
+                    result = orchestrate.main()
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)